src/main/java/au/org/ala/biocache/service/DownloadService.java

```diff
@@ -269,6 +269,9 @@
     @Value("${download.qualityFiltersTemplate:classpath:download-email-quality-filter-snippet.html}")
     public Resource downloadQualityFiltersTemplate;
 
+    @Value("${download.date.format:yyyy-MM-dd}")
```

Comment:
set default format to `dow mon dd hh:mm:ss zzz yyyy` to keep consistent with default format.

Reply:
Change made after merge

Reply:
Hi,

I set the default value to yyyy-MM-dd since it was the default hardcoded value (to avoid changes)
I agree with you to have a single datetime format accross the application.

Cheers,
Sylvain